### DISCUSSION TITLE
专业版变免费版

### DISCUSSION THREAD

**Discussion by jjsun894161** [getlantern/lantern-forum-cn#368] [General]
![付款记录](https://github.com/getlantern/lantern-forum-cn/assets/165773390/d34e5c1d-18c2-4028-8689-f73d104771d5)
专业版变免费版，已多次邮件沟通未解决问题，请发放激活码。付款记录见附件，邮箱jjsun@ntu.edu.cn。

**Comment by MaxMKMax** [comment]
@jjsun894161 您好! 好的我会帮您直接对接付款团队处理. 请您等一等.
非常抱歉因为系统上出了一些问题, 我们正在修复了. 
我们暂时会通过手动帮您升级或发送激活码. 
非常抱歉也谢谢您支持蓝灯!

**Reply by MaxMKMax** [reply]
@jjsun894161 您好. 给您发激活码了. 请查看邮件/ Jitbit. 谢谢.
